nion/instrumentation/DriftTracker.py

```diff
@@ -91,66 +202,97 @@ class DriftTracker:
     An extension to this class would be to separate the drift algorithm into its own class and allow it to be
     configured.
     """
-    def __init__(self) -> None:
+    def __init__(self, *, max_history_size: typing.Optional[int] = 10, native_axis: str="StageAxis") -> None:
```

Comment:
Consider using `native_axis: typing.Optional[str] = None` and then `native_axis = native_axis or "StageAxis"` in the function body. I think it would give us more flexibility in the future.